e2e/utils/unique/config.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Do you need utils/unique at all? I can't find any references to all these files

Reply:
deleted